### DISCUSSION TITLE
Controlnet not doing anything in A111

### DISCUSSION THREAD

**Discussion by YohanZeta** [Mikubill/sd-webui-controlnet#1566] [General]
Not really sure how to solve this problem. Basically I can't get it to do anything even with the 'Enable" box checked off. Doesn't matter which ControlType, preprossesor, or model that I select. Additionally, the 'Allow Preview' selection doesn't do anything either. The generated image output will always act as though Controlnet does not exist, even if it says it's using it in the message below the generated image.

This was the case with an older version and the current version I'm running. ( ControlNet v1.1.212 )

I don't recall seeing this in my terminal in previous attempts so I'm not sure if it's a new issue that I'm experiencing that happens to mirror the same type of issue from when I attempted to use Controlnet when it was the previous version or if it's the same problem just with new error message since I've added things to A111.
Error Messages: https://pastebin.mozilla.org/pMJsoKcM

I've attempted to uninstall and reinstall Controlnet and made sure the models were in the right directory.  Not really sure how to move forward from here.

![Screenshot 2023-06-06 003518](https://github.com/Mikubill/sd-webui-controlnet/assets/135
![Screenshot 2023-06-06 004618](https://github.com/Mikubill/sd-webui-controlnet/assets/135688172/e2f772c0-8d1a-455b-b4e0-e3af66d40fd3)
688172/adb02ff7-236
![Screenshot 2023-06-06 005235](https://github.com/Mikubill/sd-webui-controlnet/assets/135688172/5c4670c9-aec5-4b9a-aeb4-590da771d75b)
7-4fc7-93dc-5d896a9b6627)
![Screenshot 2023-06-06 004746](https://github.com/Mikubill/sd-webui-controlnet/assets/135688172/1b6d7924-087a-4181-a139-6e61d31a82c4)

![Screenshot 2023-06-06 010025](https://github.com/Mikubill/sd-webui-controlnet/assets/135688172/fbf51897-0600-48c3-b563-838e34677e2e)

I've also attempted to 'correct' the paths in the ControlNet settings but the same issue with no changes still persists. 

![Screenshot 2023-06-06 010433](https://github.com/Mikubill/sd-webui-controlnet/assets/135688172/f57b12c1-f784-4c8c-9b34-c9f9c24b9310)

**Comment by lllyasviel** [comment]
Config mismatch.

We recommend beginners to only follow the official instruction in the Readme to download/install models.

Only advanced users who know very well what they are doing should manually manage all models/configs and carefully match their filenames one-by-one on their own (for example using controlnets downloaded from other sources/civitai).
